packages/kbn-shared-ux-components/src/page_template/page_template_inner.test.tsx

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { shallow } from 'enzyme';
+import { KibanaPageTemplateInner } from './page_template_inner';
+import React from 'react';
+import { EuiEmptyPrompt, EuiPageTemplate } from '@elastic/eui';
```

Comment:
nit: we need to get into the habit of sorting imports, with newlines between classifications.  My preference is as follows, but yours may be different:

```
// imports from npm packages
import React from 'react';
import { shallow } from 'enzyme';

// imports from elastic packages
import { EuiEmptyPrompt, EuiPageTemplate } from '@elastic/eui';

// imports from relative directories
import { FooBar} from '../../../components/foo/bar';

// imports from immediate files
import { KibanaPageTemplateInner } from './page_template_inner';
```